### PR TITLE
fix(blocks): resolve on file input dialog cancel

### LINKED ISSUE #6949: Add images button remains disabled when select file dialog was cancelled
This happens on the systems with no File System Access API supported.

https://github.com/toeverything/blocksuite/assets/26658144/e246f47d-4b51-4b11-a1c2-0208363c190d

### PR DESCRIPTION
**Pull request by shvixxl**
Close #6949.

https://github.com/toeverything/blocksuite/assets/26658144/fcfbb821-70e4-4e17-89cd-7e8bb65672b2

### CODE REVIEW

File: packages/blocks/src/_common/utils/filesys.ts [hunk 1]
```diff
@@ -201,6 +201,10 @@ export async function openFileOrFiles({
       }
       resolve(input.files[0]);
     });
+    // The `cancel` event fires when the use cancels the dialog.
```
Comment (shvixxl) [inline packages/blocks/src/_common/utils/filesys.ts:204]:
```suggestion
    // The `cancel` event fires when the user cancels the dialog.
```

**Review by doodlewind — COMMENTED** [review]
Seems that there are regressions on test cases. Any idea fixing it? Thanks!

**Comment by shvixxl** [comment]
hmmm.. deployment fails when I do rebase locally, while if I do it here it succeeds.

**Comment by shvixxl** [comment]
Okay, no, it fails randomly. Is there a problem with the pull request?

**Comment by doodlewind** [comment]
@shvixxl could be a flaky test introduced by this PR. You can refer to https://github.com/toeverything/blocksuite/blob/master/BUILDING.md#testing about testing.

**Review by Saul-Mirone — APPROVED** [review]

**Pull request merged**